.github/workflows/deploy_documentation.yml

```diff
@@ -4,7 +4,11 @@ name: Deploy Documentation
 on:
   workflow_dispatch:
   push:
-    branches: [ master ]
+    branches:
+      - master
+  pull_request:
```

Comment:
i dont really want to deploy the documentation on pull request; that's a weird pattern.

Reply:
Makes sense. We have a workflow in our repos, for example: https://github.com/microbiomedata/nmdc-schema/blob/main/.github/workflows/test_pages_build.yaml which allows you to preview documentation changes upon making a PR (Action creates a public URL and adds it as a comment on the PR). Let me know if you want that added as a feature to this repo?

For now I'll remove documentation deployment upon PR's. It would be useful if you were adding something like an FAQ, updating the contributor guidelines, policy updates, etc. and you wanted to see it on the website prior to a release, but if that's not a use case at the moment, no worries.